src/presentation/api/routers/subjects.py

```
@@ -0,0 +1,152 @@
+from sqlmodel import Session, select
+
+from src.infrastructure.db.models import Subject
+from src.presentation.api.schemas import SubjectCreate, SubjectRead
+from src.presentation.api.deps import get_session
+
+router = APIRouter()
+
+
+@router.get("/", response_model=List[SubjectRead])
+def list_subjects(
+	session: Session = Depends(get_session),
```

Ruff (0.14.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)